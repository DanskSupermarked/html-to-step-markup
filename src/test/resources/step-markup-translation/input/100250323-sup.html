<p>Sandkassesand / strandsand 1 m<sup>3</sup></p> <p>Dette hvide sandkassesand som gør det meget letter for børn at lege og bygge med sandet. Det skyldes sandets sammens&aelig;tning af kornstørrelserne. Denne sammens&aelig;tning gør at sandet binder meget letter sammen og gør den dermed sjovere at lege med.</p>
