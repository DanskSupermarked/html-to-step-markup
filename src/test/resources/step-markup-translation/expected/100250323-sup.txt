Sandkassesand / strandsand 1 m<sup>3</sup>

Dette hvide sandkassesand som gør det meget letter for børn at lege og bygge med sandet. Det skyldes sandets sammensætning af kornstørrelserne. Denne sammensætning gør at sandet binder meget letter sammen og gør den dermed sjovere at lege med.
